.gitignore

```diff
@@ -0,0 +1,4 @@
+*~
+__pycache__/
+file.json
+set_variable*
```

Comment:
Missing newline at the end of the file. Don't know if that matters for .gitignore files, but worth noting.

Reply:
Got it added